### DISCUSSION TITLE
I'm doing a `podman run` and it freezes.

### DISCUSSION THREAD

**Discussion by Folaht** [containers/podman#14541] [Q&A] [answered]
I've got a pod of two containers, not counting podman pause.

```
$ sudo podman ps -a
CONTAINER ID  IMAGE                                          COMMAND     CREATED        STATUS                    PORTS                                        NAMES
7475e413455a  localhost/podman-pause:4.1.0-1652106728                    7 minutes ago  Up 7 minutes ago          192.168.178.26:12000-12002->12000-12002/udp  5ca473f4f83e-infra
f69f99b88cf5  ghcr.io/safenetwork-community/safenode:latest              7 minutes ago  Up 7 minutes ago          192.168.178.26:12000-12002->12000-12002/udp  root_node
734be703ac75  ghcr.io/safenetwork-community/safenode:latest              6 minutes ago  Exited (0) 5 minutes ago                                               join_node_1
```

Seeing one of the containers exit quickly i want to figure out what's going wrong.
So I first do a commit.
```
sudo podman commit join_node_1 debug/joinnode-ipv4
```
And then run the image.
```
sudo podman run --entrypoint /bin/bash --pod node_pod -it --rm --name debug_node debug/joinnode-ipv4
```
However, the container then fully freezes. Nothing happens.  
I can't do any debugging this way.  

What can I do to fix this issue and why is this happening?

[update]

Although for some reason it no longer happens, in between tries I did recieve a weird error..

```
sudo podman run --pod node_pod debug/joinnode-ipv4
Error: error parsing container port: invalid port number: strconv.Atoi: parsing "od": invalid syntax
```

**Comment by mheon** [comment]
Freezes in what way? Do Podman commands like `podman ps` still work?

**Comment by Folaht** [comment, marked as answer]
No, but for some reason it started working again.  
Not sure what caused it, but the only thing I can think of having done in between tries was deleting untagged podman images.
Will file this as answered until it comes up again.  

While trying out different ways to start the container, I did recieve an error once,  
so I added it in the main question.
